ocs_ci/ocs/resources/pod.py

```diff
@@ -2166,15 +2166,18 @@ def wait_for_new_osd_pods_to_come_up(number_of_osd_pods_before):
         logger.warning("None of the new osd pods reached the desired status")
 
 
-def get_pod_restarts_count(namespace=config.ENV_DATA["cluster_namespace"]):
+def get_pod_restarts_count(
+    list_of_pods=[], namespace=config.ENV_DATA["cluster_namespace"]
+):
```

Comment:
I have two points to this change.
You are changing the position of the original `namespace` parameter, so you have to review all the occurrences/usages of  this function and make sure, that if there is provided the `namespace` parameter, that it is passed with the `namespace` name, otherwise the value will be wrongly passed to the new parameter. If I didn't miss anything, I found the usage of this function on following 4 places:
* [ocs_ci/ocs/flowtest.py#L100](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/flowtest.py#L100-L102)
* [ocs_ci/ocs/flowtest.py#L121](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/flowtest.py#L121-L123)
* [tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py#L191](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py#L191)
* [tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py#L234](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py#L234)

And second suggestion is to not use the empty array as default parameter, but rather `None` (and assign empty array based on `None` value inside the function). See for example this explanation: https://www.geeksforgeeks.org/use-mutable-default-value-as-an-argument-in-python/